core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -232,7 +232,7 @@ object LogConfig {
     }
   }
 
-  private val configDef: LogConfigDef = {
+  val configDef: LogConfigDef = {
```

Comment:
We should use `private[log]` as the access modifier instead of making everything public (I assume the modifier change was for access in tests).